src/main/groovy/com/github/spotbugs/snom/internal/SpotBugsRunnerForWorker.java

```diff
@@ -96,23 +97,21 @@ public void execute() {
           findBugs2.execute();
           if (findBugs2.getErrorCount() > 0) {
             throw new GradleException(
-                "Verification failed: SpotBugs error found: "
-                    + findBugs2.getErrorCount()
-                    + ". "
+                findBugs2.getErrorCount()
+                    + " SpotBugs errors were found. "
                     + buildMessageAboutReport());
           } else if (findBugs2.getBugCount() > 0) {
             throw new GradleException(
-                "Verification failed: SpotBugs violation found: "
-                    + findBugs2.getBugCount()
-                    + ". "
+                findBugs2.getBugCount()
+                    + " SpotBugs violations were found. "
                     + buildMessageAboutReport());
           }
         }
       } catch (GradleException e) {
         if (params.getIgnoreFailures().getOrElse(Boolean.FALSE).booleanValue()) {
           final boolean showStackTraces =
               params.getShowStackTraces().getOrElse(Boolean.TRUE).booleanValue();
-          log.warn("SpotBugs reported failures", showStackTraces ? e : e.getMessage());
+          log.warn(e.getMessage(), showStackTraces ? e : null);
```

Comment:
If you are going to the trouble of redoing the output message I would like to see this changed from a ternary operator to something cleaner. 

Something along the line of 
```
@Override
    public void execute() {
      SpotBugsWorkParameters params = getParameters();
      String[] args = params.getArguments().get().toArray(new String[0]);
      DetectorFactoryCollection.resetInstance(new DetectorFactoryCollection());

      try {
        edu.umd.cs.findbugs.Version.printVersion(false);
        try (FindBugs2 findBugs2 = new FindBugs2()) {
          TextUICommandLine commandLine = new TextUICommandLine();
          FindBugs.processCommandLine(commandLine, args, findBugs2);
          findBugs2.execute();
          
          StringBuilder message = new StringBuilder();
          if (findBugs2.getErrorCount() > 0) {message.append(findBugs2.getErrorCount()).append(" SpotBugs errors were found. ");} 
          if (findBugs2.getBugCount() > 0) {message.append(findBugs2.getBugCount()).append(" SpotBugs violations were found. ");}
          if(message.length() > 0)
          {
            buildMessageAboutReport(message);
            GradleException e = new GradleException(message.toString());
            
            if(params.getIgnoreFailures().getOrElse(Boolean.FALSE)) {
              log.warn(message.toString());
              if(params.getShowStackTraces().getOrElse(Boolean.TRUE)) {
                log.warn("", e);
              }
            }
            else {
              throw e; //fail the gradle build
            }
          } 
              
        }
      } catch (Exception e) {
        throw new GradleException("Verification failed: SpotBugs execution thrown exception", e);
      }
      
    }

    private void buildMessageAboutReport(StringBuilder message) {
      String reportPath = findReportPath();
      if (reportPath != null) {
        message.append("SpotBugs report can be found in ").append(reportPath);
      }
    }

    @CheckForNull
    private String findReportPath() {
      List<String> arguments = getParameters().getArguments().get();
      int outputFileParameterIndex = arguments.indexOf("-outputFile");
      if (outputFileParameterIndex > 0) {
        return arguments.get(outputFileParameterIndex + 1);
      }
      return null;
    }
```